src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -3129,10 +3136,39 @@ private void AppendLineToOutputFunction(string name, string js, StringBuilder wr
                 return;
             }
 
+            switch (js)
+            {
+                case string s when s.StartsWith("includes.document("):
+                    HandleInclude(name, s, "let _ = RavenQuery.Include<T>(Expression<Func<T, string>> path)");
+                    break;
+                case string s when s.StartsWith("includes.timeseries("):
+                    HandleInclude(name, s, "let _ = RavenQuery.IncludeTimeSeries(T documentInstance, string timeSeriesName)");
+                    break;
+                case string s when s.StartsWith("includes.counters("):
+                    HandleInclude(name, s, "let _ = RavenQuery.includeCounters(T documentInstance, string CounterName)");
+                    break;
+                default:
+                    _declareBuilder ??= new StringBuilder();
```

Comment:
this is almost same code as `AppendIncludeFunctionBody` can we reuse it?

Reply:
done